### PR TITLE
Added Copy Group in Mapping Client

### PR DESCRIPTION
**Pull request by smmr-dn**
Added new method for Copy Group in Mapping Client. Added new base path for Grouping and Mapping (https://api.bentley.com/grouping-and-mapping/)

### CODE REVIEW

File: src/common/OperationsBase.ts [hunk 1]
```diff
@@ -35,8 +45,7 @@ export class OperationsBase {
       Accept: ACCEPT,
     };
     if (content) {
-      header["Content-Type"] = "application/json",
-      request.body = content;
+      (header["Content-Type"] = "application/json"), (request.body = content);
```
Comment (MindaugasButkus) [inline src/common/OperationsBase.ts:48]: What happened here?

File: src/common/OperationsBase.ts [hunk 2]
```diff
@@ -1,20 +1,26 @@
 /*---------------------------------------------------------------------------------------------
```
Comment (MindaugasButkus) [inline src/common/OperationsBase.ts:1]: Why are there so many changes in this file that are not related to the thing you are adding? Revert all the unnecessary changes.

Comment (MindaugasButkus) [reply src/common/OperationsBase.ts:1]: Revert all the unnecessary in all files. It's really hard to see what you actually changed.

**Comment by smmr-dn** [comment]
Commited with another account
